src/modules/fundingManager/bondingCurve/FM_BC_Bancor_Redeeming_VirtualSupply_v1.sol

```diff
@@ -541,13 +541,26 @@ contract FM_BC_Bancor_Redeeming_VirtualSupply_v1 is
         internal
         override(VirtualIssuanceSupplyBase_v1)
     {
+        uint decimalDifference;
+        uint decimals = issuanceTokenDecimals;
+
+        if (decimals <= 18) {
+            // If issuance token has 18 or fewer decimals. Ensure that decimal difference is at least 1
+            // to end up with minRequiredSupply of 10
+            decimalDifference = 18 - decimals == 0 ? 1 : 18 - decimals;
```

Comment:
Why 18 - decimals? If it is less than 18 then it can always be just 0 no (so the condition will be _virtualSupply < 1)

Reply:
Not sure whether you are asking why we go with `18` specifically (which is what the BancorFormula calculates with) or why we calculate `18 - decimals`, which is just to get the difference between the issuance tokens decimals and 18 decimals.

Also not sure what you mean exactly with "_it can always be just 0_". To arrive at the condition of `_virtualSupply < 1`, the `decimalDifference` would need to be `0`. I can not see any way for the `decimalDifference` to be `0` though, as we covered exactly that. The minimum would be `1`, even if the `decimals` are `18`, which leads to a condition of `_virtualSupply < 10`, which is exactly what we want.

Could elaborate or clarify this @ben-kaufman? 🙏

Reply:
Sorry, I think my last comment was incorrect. Looks fine to me.

Reply:
We did some back and forth around this, after which we conclude that you are right. Code has been adapted accordingly and the commit can be found here
https://github.com/InverterNetwork/contracts/pull/644/commits/d7849b3bcd7d9b90253ddbce701e57f96a42020a

@ben-kaufman

Reply:
Looks good. I think it makes more sense this way.